### PR TITLE
gpui: Activate window on Linux

### PR DESCRIPTION
**Pull request by Bloeckchengrafik**
Release Notes:
- N/A

Hello everyone,
it's me again! This is another todo!(linux) thing in gpui.

Best Regards,
Christian Bergschneider

### CODE REVIEW

**Review by mikayla-maki — APPROVED** [review]
Thank you!

File: crates/gpui/src/platform/linux/window.rs [hunk 1]
```diff
@@ -17,7 +17,10 @@ use std::{
     rc::Rc,
     sync::{self, Arc},
 };
-use xcb::{x, Xid as _};
+use xcb::{
+    x::{self, StackMode},
+    Xid as _,
```
Comment (istudyatuni) [inline crates/gpui/src/platform/linux/window.rs:22]: Why is this imported as wildcard?

Comment (Bloeckchengrafik) [reply crates/gpui/src/platform/linux/window.rs:22]: multi-line? cargo fmt said so...
StackMode separately from x itself? to access StackMode without the x:: prefix in the function.

In my opinion, it doesn't really matter if I import it separately or use the prefix, 
and (the most important reason) my ide said so. 

Do you think this is bad? I'll open a new pr if you say it's worth it!

Comment (istudyatuni) [reply crates/gpui/src/platform/linux/window.rs:22]: Sorry, I wasn't clear enough. My question is about `Xid as _` (I don't know how it's called. Placeholder use?) It seems like this import is unused.

Comment (istudyatuni) [reply crates/gpui/src/platform/linux/window.rs:22]: I found an answer https://doc.rust-lang.org/reference/items/use-declarations.html#underscore-imports

**Pull request merged**